Source/d_player.h

```diff
@@ -74,7 +74,9 @@ typedef enum
   // Not really a cheat, just a debug aid.
   CF_NOMOMENTUM       = 4,
   // [FG] FPS counter widget
-  CF_SHOWFPS          = 8
+  CF_SHOWFPS          = 8,
```

Comment:
SHOWFPS could as well get moved to the `powers[]` array. What do you think?

Reply:
Ok, should I do it in this PT?

Reply:
Ok.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/448/commits/b3eaf9488bf57240dd9487f8be7ad57e1abef863